apps/src/lib/ui/ParentLetter.jsx

```diff
@@ -0,0 +1,237 @@
+          involved.
+        </p>
+        <h1>Step 1 - Encourage your child, show interest</h1>
+        <p>
+          One of the best ways to show your interest is to ask your child to
+          explain what they’re learning and show you a project they are proud of
+          (<a href={ENGAGEMENT_URL}>details on how to engage your child</a>
+          ).
+        </p>
+        <h1>Step 2 - Get your child set up to use Code.org at home</h1>
+        <SignInInstructions loginType={loginType} sectionCode={sectionCode} />
+        <p>
+          At the top of their homepage, your student can continue the course
+          they are doing with their classroom at school. They can also create
+          their own{' '}
+          <a href={PROJECT_GALLERY_URL}>
+            games or artwork in the Project Gallery
+          </a>{' '}
+          or check out <a href={AT_HOME_URL}>code.org/athome</a> for ideas for
+          things to work on at home.
+        </p>
+        <h1>Step 3 - Connect your email to your student’s account</h1>
+        <p>
+          Keep up to date with what your student is working on and receive
+          updates from Code.org.
+        </p>
+        <ol>
+          <li>Have your child sign in to Code.org</li>
+          <li>
+            Click on the User Menu in the top right corner of the site, then
+            click on Account Settings.
+          </li>
+          <li>
+            Scroll down to the section “For Parents and Guardians” and add your
+            email address.
+          </li>
+        </ol>
+        <h1>Step 4 - Review Code.org’s privacy policy</h1>
+        <p>
+          Code.org assigns utmost importance to student safety and security.
+          Code.org has signed the{' '}
+          <a href={PRIVACY_PLEDGE_URL}>Student Privacy Pledge</a> and their
+          privacy practices have received{' '}
+          <a href={COMMON_SENSE_ARTICLE_URL}>
+            one of the highest overall scores from Common Sense Media
+          </a>
+          . You can find further details by viewing Code.org’s{' '}
+          <a href={PRIVACY_POLICY_URL}>Privacy Policy</a>.
+        </p>
+        <p>
+          Computer science teaches students critical thinking, problem solving,
+          and digital citizenship, and benefits all students in today’s world,
+          no matter what opportunities they pursue in the future.
+        </p>
+        <p>
+          Please let me know if you have any questions and thank you for your
+          continued support of your child and of our classroom!
+        </p>
+        <p>{teacherName}</p>
+      </article>
+    </div>
+  );
+}
+ParentLetter.propTypes = {
+  loginType: PropTypes.oneOf(Object.values(SectionLoginType)).isRequired,
+  sectionCode: PropTypes.string,
+  teacherName: PropTypes.string.isRequired
+};
+
+const Header = () => {
+  return (
+    <header style={{backgroundColor: color.teal, marginBottom: 30}}>
```

Comment:
nit - our styleguide prefers extracting these to a style object rather than doing it inline.

Reply:
:+1: Will update.

Independent of this PR, would love to have a conversation at some point about whether people actually like this rule: It's one [we introduced without much discussion](https://github.com/code-dot-org/code-dot-org/pull/7339) and I mostly find it annoying now.  It makes composing styles a little unintuitive. I'm also not sure it's enforceable by any existing linter rule, and in general we try to cut things from our styleguide that we can't automatically enforce.  What are your thoughts on it?